connector/checkpoint/checkpoint_view.xml

```diff
@@ -48,6 +48,7 @@
                         delete="false" edit="false" version="7.0">
                     <field name="model_id"/>
                     <field name="record"/>
+                    <field name="message"/>
```

Comment:
Reason I said something is that my checkpoints screen is already essentially full on a 1200px screen:

![image](https://cloud.githubusercontent.com/assets/7397578/19734633/248c797c-9b5d-11e6-94b5-adf1fe20e40c.png)


I won't block though, as I don't have another solution and I can see the use case in your's.